lib/movie_header.dart

```diff
@@ -0,0 +1,54 @@
+import 'movie.dart';
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+class MovieHeader extends StatelessWidget {
+  const MovieHeader({super.key, required this.movie});
+
+  final Movie movie;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(children: [
+      Image(
+        image: AssetImage(
+          movie.backdrop,
+        ),
+        fit: BoxFit.cover,
+      ),
+      Positioned(
+        bottom: UIConstants.titlePositionBottom,
+        left: UIConstants.titlePositionLeft,
+        right: UIConstants.titlePositionRight,
+        child: Container(
+          height: UIConstants.titleContainerHeight,
+          decoration: BoxDecoration(
+            gradient: LinearGradient(
+              colors: [
+                Colors.transparent,
+                Colors.black
+                    .withOpacity(UIConstants.titleContainerOpacity),
+              ],
+              begin: Alignment.topCenter,
+              end: Alignment.bottomCenter,
+            ),
+          ),
+          child: Center(
+            child: Text(
+              movie.title,
+              style: UIConstants.titleTextStyle,
+            ),
+          ),
+        ),
+      ),
+    ]);
+  }
+}
+
+
+
+
+
+
+
+
```

Comment:
Remove these extra new lines

Reply:
I don't know why you see these extra lines, my file (at least how I see it in AndroidStudio) finishes at line 48!